app/bootstrap.php

```diff
@@ -31,11 +31,9 @@
 		'line' => $e->getLine(),
 		'stack_trace' => $e->getTraceAsString()
 	] );
-	return new JsonResponse(
-		[
-			'message' => $e->getMessage(),
-			'code' => $code
-		],
+
```

Comment:
@KaiNissen this thing works for all routes, so no need to do anything route specific